tests/foreman/api/test_contentviewfilter.py

```diff
@@ -212,36 +227,41 @@ def test_positive_create_with_repo(self):
         :CaseLevel: Integration
         """
         cvf = entities.RPMContentViewFilter(
-            content_view=self.content_view, inclusion=True, repository=[self.repo]
+            content_view=content_view,
+            inclusion=True,
+            repository=[sync_repo],
         ).create()
-        self.assertEqual(cvf.repository[0].id, self.repo.id)
+        assert cvf.repository[0].id == sync_repo.id
 
     @pytest.mark.tier2
-    def test_positive_create_with_original_packages(self):
+    @pytest.mark.parametrize('original_packages', **parametrized((True, False)))
```

Comment:
```suggestion
    @pytest.mark.parametrize('original_packages', [True, False])
```
same suggestion for rest of PR code, wherever applicable.

Reply:
Done.